--- kdeutils-3.0.5a/kab/kab_mkIII/Makefile.in.orig	Fri Feb  7 21:43:51 2003
+++ kdeutils-3.0.5a/kab/kab_mkIII/Makefile.in	Fri Feb  7 21:55:33 2003
@@ -916,11 +916,11 @@
 	rm -f libkab3part_la.all_cpp.final libkab3part_la.all_cpp.files
 
 #>+ 11
-kab3.all_cpp.cpp: $(srcdir)/Makefile.in $(srcdir)/main.cpp $(srcdir)/kab3.cpp kab3viewcontainer.moc.cpp  kab3.moc
+kab3.all_cpp.cpp: $(srcdir)/Makefile.in $(srcdir)/main.cpp $(srcdir)/kab3.cpp kab3viewcontainer.moc.cc kab3.moc
 	@echo 'creating kab3.all_cpp.cpp ...'; \
 	rm -f kab3.all_cpp.files kab3.all_cpp.final; \
 	echo "#define KDE_USE_FINAL 1" >> kab3.all_cpp.final; \
-	for file in main.cpp kab3.cpp kab3viewcontainer.moc.cpp ; do \
+	for file in main.cpp kab3.cpp kab3viewcontainer.moc.cc ; do \
 	  echo "#include \"$$file\"" >> kab3.all_cpp.files; \
 	  test ! -f $(srcdir)/$$file || egrep '^#pragma +implementation' $(srcdir)/$$file >> kab3.all_cpp.final; \
 	done; \
