--- kdeutils/kdepasswd/userinfo/userinfo.cpp.orig	2004-01-17 10:47:41.000000000 +0100
+++ kdeutils/kdepasswd/userinfo/userinfo.cpp	2004-01-17 11:14:14.000000000 +0100
@@ -284,9 +284,9 @@
   // Get face dirs
   m_facesDir =  KGlobal::dirs()->resourceDirs("data").last() + "kdm/" + KDM_FACES_DIR;
 
-  KConfig *kdmconfig = new KConfig("kdm/kdmrc", true);
+  KConfig *kdmconfig = new KConfig("/etc/X11/kdm/kdmrc", true);
   kdmconfig->setGroup("X-*-Greeter");
-  m_userPicsDir = kdmconfig->readEntry( "FaceDir", KGlobal::dirs()->resourceDirs("data").last() + "kdm/faces" ) + '/';
+  m_userPicsDir = kdmconfig->readEntry( "FaceDir", "/etc/X11/kdm/faces" ) + '/';
   QString fs = kdmconfig->readEntry( "FaceSource" );
   if (fs == QString::fromLatin1("UserOnly"))
     m_FaceSource = FACE_SRC_USERONLY;
