--- kdeutils-3.2.3/ksim/ksimview.cpp.orig	2004-05-30 16:29:20.000000000 +0200
+++ kdeutils-3.2.3/ksim/ksimview.cpp	2004-06-09 12:18:58.798814560 +0200
@@ -272,12 +272,14 @@
   for (it = list.begin(); it != list.end(); ++it) {
     if ((*it).isDifferent()) {
       if ((*it).isEnabled()) { // Go through the added/removed plugins and load/unload them
-        addPlugin((*it).filename());
+        const KDesktopFile kdf((*it).filename());
+        addPlugin(kdf);      
         m_prefDialog->createPage((*it).libName());
       }
       else {
         m_prefDialog->removePage((*it).libName());
-        removePlugin((*it).filename());
+	const KDesktopFile kdf((*it).filename());
+	removePlugin(kdf);
       }
     }
   }
@@ -304,7 +306,8 @@
   QStringList locatedFiles = KGlobal::dirs()->findAllResources("data", "ksim/monitors/*.desktop");
   QStringList::ConstIterator it;
   for (it = locatedFiles.begin(); it != locatedFiles.end(); ++it)
-    addPlugin(*it);
+    const KDesktopFile kdf(*it, true);
+    addPlugin(kdf);
 }
 
 void KSim::MainView::addPlugin(const QString& _file, bool force)
