--- kdeutils-3.0.0/kedit/ktextfiledlg.cpp.confirmOverwrite	Sat Oct 13 06:22:57 2001
+++ kdeutils-3.0.0/kedit/ktextfiledlg.cpp	Wed Apr  3 15:33:16 2002
@@ -1,4 +1,7 @@
+#include <iostream>
+using namespace std;
 // -*- c++ -*-
+// vim:ts=2:sw=2
 /* This file is part of the KDE libraries
     Copyright (C) 2001 Wolfram Diestel <wolfram@steloj.de>
  
@@ -28,6 +31,7 @@
 #include <kdiroperator.h>
 #include <krecentdocument.h>
 #include <ktoolbar.h>   
+#include <kmessagebox.h>
 
 #include "ktextfiledlg.h"
 
@@ -182,16 +186,25 @@
   dlg.setCaption(caption.isNull() ? i18n("Save As") : caption);
   dlg.setKeepLocation( true );
   
-  dlg.exec();
-  
-  KURL url = dlg.selectedURL();
-  if (!url.isMalformed()) {
-    if ( url.isLocalFile() )
-      KRecentDocument::add( url.path(-1) );
-    else
-      KRecentDocument::add( url.url(-1) );
-  }
-
+  bool repeat;
+	KURL url;
+  do {
+		repeat=false;
+    dlg.exec();
+  
+    url = dlg.selectedURL();
+    if (!url.isMalformed()) {
+      if ( url.isLocalFile() ) {
+        if(QFile::exists(url.path(-1))) {
+          if(KMessageBox::questionYesNo(parent, i18n("Overwrite existing file %1?").arg(url.fileName(true)))==KMessageBox::No)
+            repeat=true;
+        } else
+          KRecentDocument::add( url.path(-1) );
+      } else
+        KRecentDocument::add( url.url(-1) );
+    }
+  } while(repeat);
+    
   // append encoding to the URL params
   url.setFileEncoding(dlg.encoding());
 
