--- kdeutils-030317/khexedit/khexedit.desktop~	Thu Mar  6 01:06:46 2003
+++ kdeutils-030317/khexedit/khexedit.desktop	Thu Mar 20 14:10:54 2003
@@ -67,4 +67,4 @@
 Terminal=0
 X-KDE-StartupNotify=true
 X-DCOP-ServiceType=Multi
-Categories=Qt;KDE;Utility;X-KDE-More
+Categories=Qt;KDE;X-HexEditor
--- kdeutils-030726/kgpg/kgpg.desktop~	2003-07-21 00:59:10.000000000 +0200
+++ kdeutils-030726/kgpg/kgpg.desktop	2003-07-27 02:29:24.000000000 +0200
@@ -14,3 +14,4 @@
 X-KDE-autostart-after=panel
 X-KDE-StartupNotify=false
 X-KDE-autostart-condition=kgpgrc:User Interface:AutoStart:false
+Categories=Qt;KDE;Utility
