--- kdeutils-3.0.3/charselectapplet/Makefile.am.orig	Fri Mar  8 21:52:08 2002
+++ kdeutils-3.0.3/charselectapplet/Makefile.am	Tue Sep 24 23:09:09 2002
@@ -12,7 +12,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-kcharselectapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+kcharselectapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 kcharselectapplet_la_LIBADD = $(LIB_KDEUI)
 
 messages:
--- kdeutils-3.0.3/charselectapplet/Makefile.in.orig	Wed Aug  7 20:25:03 2002
+++ kdeutils-3.0.3/charselectapplet/Makefile.in	Tue Sep 24 23:10:22 2002
@@ -241,7 +241,7 @@
 
 EXTRA_DIST = $(lnk_DATA)
 
-kcharselectapplet_la_LDFLAGS = $(all_libraries) -version-info 1:0:0 -module -no-undefined
+kcharselectapplet_la_LDFLAGS = $(all_libraries) -module -no-undefined -avoid-version
 kcharselectapplet_la_LIBADD = $(LIB_KDEUI)
 subdir = charselectapplet
 mkinstalldirs = $(SHELL) $(top_srcdir)/admin/mkinstalldirs
