--- kdeutils-030329/kdepasswd/userinfo/userinfo.cpp~	Mon Mar 24 21:19:25 2003
+++ kdeutils-030329/kdepasswd/userinfo/userinfo.cpp	Wed Apr  2 02:15:53 2003
@@ -280,11 +280,11 @@
 void KUserInfoConfig::load()
 {
   // Get face dirs
-  QString m_kdmDir = KGlobal::dirs()->resourceDirs("data").last() + "kdm/";
+  QString m_kdmDir = "/etc/X11/kdm";
   m_userPicsDir = m_kdmDir + KDM_USER_FACES_DIR;
-  m_facesDir = m_kdmDir + KDM_FACES_DIR;
+  m_facesDir = KGlobal::dirs()->resourceDirs("data").last() + "kdm/pics/faces/";
 
-  KConfig *kdmconfig = new KConfig("kdm/kdmrc", true);
+  KConfig *kdmconfig = new KConfig( "/etc/X11/kdm/kdmrc", true );
   kdmconfig->setGroup("X-*-Greeter");
   QString fs = kdmconfig->readEntry( "FaceSource" );
   if (fs == QString::fromLatin1("UserOnly"))
