--- kdeutils-3.2.0/ksim/ksimview.cpp.orig	2003-10-26 11:58:52.000000000 +0100
+++ kdeutils-3.2.0/ksim/ksimview.cpp	2004-02-18 12:56:08.000000000 +0100
@@ -272,12 +272,14 @@
   for (it = list.begin(); it != list.end(); ++it) {
     if ((*it).isDifferent()) {
       if ((*it).isEnabled()) { // Go through the added/removed plugins and load/unload them
-        addPlugin(KDesktopFile((*it).filename()));
+        const KDesktopFile kdf((*it).filename());
+        addPlugin(kdf);
         m_prefDialog->createPage((*it).libName());
       }
       else {
         m_prefDialog->removePage((*it).libName());
-        removePlugin(KDesktopFile((*it).filename()));
+        const KDesktopFile kdf((*it).filename());
+        removePlugin(kdf);
       }
     }
   }
@@ -304,7 +306,10 @@
   QStringList locatedFiles = KGlobal::dirs()->findAllResources("data", "ksim/monitors/*.desktop");
   QStringList::ConstIterator it;
   for (it = locatedFiles.begin(); it != locatedFiles.end(); ++it)
-    addPlugin(KDesktopFile(*it, true));
+  {
+    const KDesktopFile kdf(*it, true);
+    addPlugin(kdf);
+  }
 }
 
 void KSim::MainView::addPlugin(const KDesktopFile &file, bool force)
