Index: charselectapplet/kcharselectapplet.desktop
===================================================================
--- charselectapplet/kcharselectapplet.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ charselectapplet/kcharselectapplet.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -89,6 +89,7 @@
 Comment[sr]=Бира стране и специјалне знакове за клипборд
 Comment[sr@Latn]=Bira strane i specijalne znakove za klipbord
 Comment[sv]=Välj främmande tecken och specialtecken för klippbordet
+Comment[tr]=Yabancı ve özel karakterleri panoya kopyalar
 Comment[uk]=Вибрати і вставити в кишеню іноземні та спеціальні символи
 Comment[uz]=Ҳарф ва махсус белгиларни хотирага олиш воситаси
 Comment[zh_CN]=将外语和特殊字符拾取到剪贴板
Index: kfloppy/floppy_format.desktop
===================================================================
--- kfloppy/floppy_format.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kfloppy/floppy_format.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -38,6 +38,7 @@
 Name[sl]=Formatiraj
 Name[sr]=Форматирај
 Name[sr@Latn]=Formatiraj
+Name[tr]=Biçimlendir
 Name[uk]=Форматування
 Name[uz]=Форматлаш
 Name[zh_CN]=格式化
Index: kfloppy/floppy.cpp
===================================================================
--- kfloppy/floppy.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kfloppy/floppy.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -702,7 +702,11 @@
 	labelnameconfig = config->readEntry( "Label", i18n("Volume label, maximal 11 characters", "KDE Floppy") );
 	quickformatconfig = config->readNumEntry("QuickFormat",0);
 	driveconfig = config->readEntry( "FloppyDrive", i18n("Primary") );
+#if defined(ANY_LINUX)
+	densityconfig = config->readEntry( "Density", i18n( "Auto-Detect" ) );
+#else
 	densityconfig = config->readEntry( "Density", i18n("3.5\" 1.44MB") );
+#endif
 	filesystemconfig = config->readEntry( "Filesystem", i18n("DOS") );
 
 }
Index: kfloppy/format.cpp
===================================================================
--- kfloppy/format.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kfloppy/format.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -269,14 +269,20 @@
 		emit status(i18n("Unexpected drive number %1.").arg(drive),-1);
 		return false;
 	}
-	if (!( /* (2880==de)  || */ (1440==density) || (720==density) ||
-		(1200==density) || (360==density)))
+
+	fdinfo *deviceinfo = fdtable;
+	for ( ; deviceinfo && (deviceinfo->devices) ; deviceinfo++)
 	{
+		if (deviceinfo->blocks != density)
+			continue;
+        }
+	if (!deviceinfo)
+	{
 		emit status(i18n("Unexpected density number %1.").arg(density),-1);
 		return false;
 	}
 
-	fdinfo *deviceinfo = fdtable;
+	deviceinfo = fdtable;
 	for ( ; deviceinfo && (deviceinfo->devices) ; deviceinfo++)
 	{
 		if (deviceinfo->blocks != density)
Index: kgpg/kgpg.desktop
===================================================================
--- kgpg/kgpg.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kgpg/kgpg.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -5,7 +5,7 @@
 Exec=kgpg %U
 Icon=kgpg
 DocPath=kgpg/index.html
-MimeType=application/pgp-encrypted;application/pgp-signature;application/pgp-keys
+MimeType=application/pgp-encrypted;application/pgp-signature;application/pgp-keys;
 GenericName=Encryption Tool
 GenericName[ar]=أداة للتشفير
 GenericName[bg]=Програма за шифроване
Index: kgpg/encryptfile.desktop
===================================================================
--- kgpg/encryptfile.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kgpg/encryptfile.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -7,6 +7,7 @@
 Name=Encrypt File
 Name[ar]=تشفير ملف
 Name[bg]=Шифроване на файл
+Name[br]=E&nrinegañ ar restr
 Name[bs]=Šifriraj datoteku
 Name[ca]=Xifra fitxer
 Name[cs]=Zašifrovat soubor
Index: ktimer/ktimer.desktop
===================================================================
--- ktimer/ktimer.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ktimer/ktimer.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -37,6 +37,7 @@
 GenericName[sr@Latn]=Lanser odbrojavanja
 GenericName[sv]=Nerräknande startprogram
 GenericName[ta]= குறிப்பு எடுப்பான் நேரக்குறைப்பான்
+GenericName[tr]=Gerisayım Çalıştırıcı
 GenericName[uk]=Таймер
 GenericName[zh_CN]=倒计时
 GenericName[zh_TW]=倒數發射計時
@@ -55,7 +56,6 @@
 Name[ta]= Kநேரம் காட்டி
 Name[tg]=KВақтсанҷ
 Name[th]=ตั้งเวลาทำงาน
-Name[tr]=Zamanlayıcı
 Name[ven]=Ratshikhathi wa K
 Name[xh]=Umjongi Wexesha
 Name[zh_TW]=KDE 計時器
Index: klaptopdaemon/applnk/.directory
===================================================================
--- klaptopdaemon/applnk/.directory	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ klaptopdaemon/applnk/.directory	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -5,48 +5,35 @@
 Name[az]=Dizüstü
 Name[bg]=Лаптоп
 Name[br]=Hezoug
-Name[ca]=Portàtil
+Name[cs]=Notebook
 Name[cy]=Gliniadur
-Name[da]=Bærbar
 Name[de]=Notebook
-Name[el]=Φορητός υπολογιστής
+Name[el]=Φορητό
 Name[eo]=Portkomputilo
 Name[es]=Portátil
-Name[eu]=Magalekoa
-Name[fi]=Kannettava
-Name[fr]=Ordinateur portable
 Name[ga]=Ríomhaire Glúine
 Name[gl]=Portátil
-Name[he]=מחשב נייד
 Name[hi]=लैपटॉप
-Name[hu]=Noteszgép
 Name[is]=Ferðavél
-Name[it]=Computer portatile
 Name[ja]=ラップトップ
+Name[km]=កុំព្យួទ័រ​យួរដៃ
 Name[ko]=랩탑
 Name[lv]=Laptops
 Name[mk]=Лаптоп
 Name[ms]=Komputer riba
-Name[nb]=Bærbar maskin
-Name[nl]=Schootcomputer
-Name[nn]=Bærbar
-Name[pa]=ਲੈਪਟੋਪ
+Name[nds]=Klappreekner
 Name[pt]=Portátil
-Name[ru]=Ноутбук
-Name[sl]=Prenosnik
+Name[pt_BR]=Portátil
 Name[sv]=Bärbar dator
 Name[ta]=  மடிக் கணினி
 Name[tg]=Компютери Дастӣ
 Name[th]=แลปทอป
-Name[tr]=Dizüstü
-Name[uk]=Мобільний комп'ютер (лептоп)
+Name[tr]=Dizüstü Bilgisayar
 Name[uz]=Лаптоп
 Name[ven]=Khomupwutha pfarwa
 Name[wa]=Poirtåve
 Name[xh]=Umphezulu osongiweyo
-Name[xx]=xxLaptopxx
-Name[zh_CN]=便携电脑
-Name[zh_TW]=膝上型的電腦
+Name[zh_TW]=筆記型電腦
 Name[zu]=Ikhomputha eyisicaba ephathwayo
 SortOrder=battery.desktop,bwarning.desktop,cwarning.desktop,power.desktop,pcmcia.desktop
 Icon=laptop_settings
Index: klaptopdaemon/applnk/klaptopdaemon.desktop
===================================================================
--- klaptopdaemon/applnk/klaptopdaemon.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ klaptopdaemon/applnk/klaptopdaemon.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -18,7 +18,7 @@
 Comment[ca]=Monitor de bateria de portàtils
 Comment[cs]=Monitor baterií laptopu
 Comment[da]=Batteriovervågning for bærbar
-Comment[de]=Benachrichtigt über den Akku-Ladezustand
+Comment[de]=Benachrichtigung über den Akku-Ladezustand
 Comment[el]=Επόπτης μπαταρίας φορητού
 Comment[es]=Monitor de batería del portátil
 Comment[et]=Sülearvuti aku monitooring
@@ -52,6 +52,7 @@
 Comment[sr@Latn]=Nadgleda bateriju laptopa
 Comment[sv]=Batterimonitor för bärbar dator
 Comment[ta]= மடிக் கணினி மின்கலம் கண்காணிப்பாளன்
+Comment[tr]=Dizüstü bilgisayarlar için pil izleyici
 Comment[uk]=Монітор батарей лептопа
 Comment[uz]=Лаптопнинг батареясини назорат қилиш
 Comment[zh_CN]=便携电脑电池监视器
Index: kedit/KEdit.desktop
===================================================================
--- kedit/KEdit.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kedit/KEdit.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Encoding=UTF-8
-MimeType=text/plain
+MimeType=text/plain;
 GenericName=Simple Text Editor
 GenericName[ar]=محرر نصوص بسيط
 GenericName[bg]=Текстов редактор
Index: kedit/ktextfiledlg.h
===================================================================
--- kedit/ktextfiledlg.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kedit/ktextfiledlg.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -60,7 +60,7 @@
 	 const QString& caption,
 	 const QString& encoding = QString::null);
 
-    QString &encoding() { return enc; };
+    QString &encoding() { return enc; }
     void setEncoding(const QString& encoding);
 
   protected slots:
Index: ksim/monitors/mail/mail.desktop
===================================================================
--- ksim/monitors/mail/mail.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/mail/mail.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -11,7 +11,7 @@
 Comment[de]=Hilfsprogramm zur Mailüberwachung
 Comment[el]=Πρόσθετο εποπτείας αλληλογραφίας
 Comment[eo]=Disk-observa kromaĵo
-Comment[es]=Plugin de monitorización de correo
+Comment[es]=Complemento de monitorización de correo
 Comment[et]=E-posti monitooring
 Comment[eu]=Postaren Plugin Begiralea
 Comment[fa]=وصلۀ نمایشگر نامه
@@ -23,7 +23,7 @@
 Comment[hu]=E-mail-figyelő bővítőmodul
 Comment[is]=Pósteftirlitsforrit
 Comment[it]=Plugin di controllo posta
-Comment[ja]=メールモニタ プラグイン
+Comment[ja]=メールモニタプラグイン
 Comment[kk]=Поштаны бақылау модулі
 Comment[km]=កម្មវិធី​ជំនួយ​របស់​កម្មវិធី​ត្រួតពិនិត្យ​សំបុត្រ
 Comment[lt]=Pašto stebėtojo priedas
Index: ksim/monitors/filesystem/filesystemstats.cpp
===================================================================
--- ksim/monitors/filesystem/filesystemstats.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/filesystem/filesystemstats.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -30,7 +30,7 @@
 #include <sys/types.h>
 #include <sys/param.h>
 
-#ifdef HAVE_SYS_STATVFS_H
+#if defined(HAVE_SYS_STATVFS_H) && !defined(__DragonFly__)
 #include <sys/statvfs.h>
 #elif defined( HAVE_SYS_STATFS_H )
 #include <sys/statfs.h>
@@ -59,10 +59,12 @@
 #include <stdio.h>
 #include <unistd.h>
 
-#ifdef HAVE_STATVFS
+#if defined(HAVE_STATVFS) && !defined(__DragonFly__)
 typedef struct statvfs ksim_statfs;
+#define ksim_getfsstat getvfsstat
 #elif defined( HAVE_STATFS ) || defined( Q_OS_FREEBSD )
 typedef struct statfs ksim_statfs;
+#define ksim_getfsstat getfsstat
 #else
 typedef struct // fall back for (possibly) non-supported systems
 {
@@ -73,7 +75,7 @@
 
 int fsystemStats( const char * file, ksim_statfs & stats )
 {
-#ifdef HAVE_STATVFS
+#if defined(HAVE_STATVFS) && !defined(__DragonFly__)
   return statvfs( file, &stats );
 #elif defined( HAVE_STATFS ) || defined( Q_OS_FREEBSD )
   return statfs( file, &stats );
@@ -162,9 +164,9 @@
 #endif
 
 #ifdef USE_FSSTAT
-  struct statfs sfs[32];
+  ksim_statfs sfs[32];
   int fs_count;
-  if ( ( fs_count = getfsstat( sfs, sizeof( sfs ), 0 ) ) != -1 )
+  if ( ( fs_count = ksim_getfsstat( sfs, sizeof( sfs ), 0 ) ) != -1 )
   {
     for ( int i = 0; i < fs_count; i++ )
     {
Index: ksim/monitors/lm_sensors/Lm_sensors.desktop
===================================================================
--- ksim/monitors/lm_sensors/Lm_sensors.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/lm_sensors/Lm_sensors.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -22,7 +22,7 @@
 Comment[hu]=Lm_sensors hardvermonitor
 Comment[is]=Eftirlit með stöðu Lm_sensors
 Comment[it]=Controllo di stato Lm_sensors
-Comment[ja]=Lm センサーの状態モニター
+Comment[ja]=Lm センサーの状態モニタ
 Comment[kk]=Lm_sensor сенсорларының күйін бақылау
 Comment[km]=កម្មវិធី​ត្រួតពិនិត្យ​ស្ថានភាព Lm_sensors
 Comment[lt]=Lm_sensors būsenos stebėtojas
Index: ksim/monitors/cpu/cpu.desktop
===================================================================
--- ksim/monitors/cpu/cpu.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/cpu/cpu.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -39,7 +39,7 @@
 Comment[de]=CPU-Nutzungsanzeige
 Comment[el]=Πρόσθετο εποπτείας KME
 Comment[eo]=Prozecilo-observila kromaĵo
-Comment[es]=Plugin de monitorización de la CPU
+Comment[es]=Complemento de monitorización de la CPU
 Comment[et]=Protsessori monitooring
 Comment[eu]=CPU Plugin Begiralea
 Comment[fa]=وصلۀ نمایشگر واحد پردازش مرکزی
@@ -51,7 +51,7 @@
 Comment[hu]=CPU-figyelő bővítőmodul
 Comment[is]=Örgjörva eftirlitsíforrit
 Comment[it]=Plugin di controllo CPU
-Comment[ja]=CPU モニタ プラグイン
+Comment[ja]=CPU モニタプラグイン
 Comment[kk]=Процессорды бақылау модулі
 Comment[km]=កម្មវិធី​ជំនួយ​នៃ​កម្មវិធី​ត្រួតពិនិត្យ​ថាស CPU Monitor
 Comment[lt]=CPU stebėtojo priedas
Index: ksim/monitors/cpu/ksimcpu.cpp
===================================================================
--- ksim/monitors/cpu/ksimcpu.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/cpu/ksimcpu.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -44,7 +44,13 @@
 
 #ifdef Q_OS_BSD4
 #include <sys/param.h>
-#ifdef Q_OS_FREEBSD
+#if defined(__DragonFly__)
+#include <kinfo.h>
+#include <err.h>
+#include <sys/time.h>
+#include <sys/resource.h>
+
+#elif defined Q_OS_FREEBSD
 #if __FreeBSD_version < 500101
 #include <sys/dkstat.h>
 #else
@@ -238,7 +244,15 @@
   fseek(m_procFile, 0L, SEEK_SET);
 #endif
 
-#ifdef Q_OS_FREEBSD
+#ifdef __DragonFly__
+  struct kinfo_cputime cp_time;
+  if (kinfo_get_sched_cputime(&cp_time))
+    err(1, "kinfo_get_sched_cputime");
+  cpu.user = cp_time.cp_user;
+  cpu.nice = cp_time.cp_nice;
+  cpu.sys = cp_time.cp_sys;
+  cpu.idle = cp_time.cp_idle;
+#elif defined(Q_OS_FREEBSD)
 #warning "add support for SMP on FreeBSD"
   Q_UNUSED(cpuNumber);
   static int name2oid[2] = { 0, 3 };
@@ -268,6 +282,9 @@
   cpu.idle = cpuTime[CP_IDLE];
 #endif
 
+#if defined(Q_OS_NETBSD)
+#define KERN_CPTIME KERN_CP_TIME
+#endif
 #if defined(Q_OS_OPENBSD) || defined(Q_OS_NETBSD)
 //#warning "add support for SMP on OpenBSD and NetBSD"
   int name2oid[2] = { CTL_KERN, KERN_CPTIME };
Index: ksim/monitors/disk/ksimdisk.cpp
===================================================================
--- ksim/monitors/disk/ksimdisk.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/disk/ksimdisk.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -41,8 +41,14 @@
 #include <progress.h>
 #include <themetypes.h>
 
-#ifdef Q_OS_FREEBSD
+#if defined(__DragonFly__)
+#include <sys/time.h>
 #include <sys/param.h>
+#include <sys/resource.h>
+#include <devstat.h>
+#include <stdlib.h>
+#elif defined(Q_OS_FREEBSD)
+#include <sys/param.h>
 #if __FreeBSD_version < 500101
 #include <sys/dkstat.h>
 #else
@@ -343,7 +349,7 @@
 #endif
 
 #ifdef Q_OS_FREEBSD
-#if __FreeBSD_version < 500107
+#if defined(__DragonFly__) || __FreeBSD_version < 500107
 #define	devstat_getdevs(fd, stats)	getdevs(stats)
 #define	devstat_selectdevs		selectdevs
 #define	bytes_read(dev)			(dev).bytes_read
Index: ksim/monitors/disk/disk.desktop
===================================================================
--- ksim/monitors/disk/disk.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/disk/disk.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -11,7 +11,7 @@
 Comment[de]=Hilfsprogramm zur Festplattenüberwachung
 Comment[el]=Πρόσθετο εποπτείας δίσκου
 Comment[eo]=Disk-observa kromaĵo
-Comment[es]=Plugin de monitorización de disco
+Comment[es]=Complemento de monitorización de disco
 Comment[et]=Ketta monitooring
 Comment[eu]=Diskoaren Plugin Begiralea
 Comment[fa]=وصلۀ نمایشگر دیسک
@@ -23,7 +23,7 @@
 Comment[hu]=Lemezfigyelő bővítőmodul
 Comment[is]=Disk eftirlitsforrit
 Comment[it]=Plugin di controllo disco
-Comment[ja]=ディスクモニタ プラグイン
+Comment[ja]=ディスクモニタプラグイン
 Comment[kk]=Дискін бақылау модулі
 Comment[km]=កម្មវិធី​ជំនួយ​នៃ​កម្មវិធី​ត្រួតពិនិត្យ​ថាស
 Comment[lt]=Disko stebėtojo priedas
Index: ksim/monitors/i8k/i8k.desktop
===================================================================
--- ksim/monitors/i8k/i8k.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/monitors/i8k/i8k.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -8,7 +8,7 @@
 Comment[cy]=Ategyn Arsylwydd Caledwedd Dell I8K
 Comment[de]=Dell I8K Hardware Überwachungs-Plugin
 Comment[el]=Πρόσθετο εποπτείας υλικού Dell I8K
-Comment[es]=Plugin de monitorización de hardware para Dell I8K
+Comment[es]=Complemento de monitorización de hardware para Dell I8K
 Comment[et]=Dell I8K riistvara monitooringu plugin
 Comment[eu]=Dell I8K Hardwarearen Plugin Begiralea
 Comment[fa]=وصلۀ نمایشگر سخت‌افزار Dell I8K
@@ -19,7 +19,7 @@
 Comment[hu]=Dell I8K hardvermonitor bővítőmodul
 Comment[is]=Íforrit til að fylgjast með Dell I8K vélbúnaði
 Comment[it]=Plugin di controllo hardware Dell I8K
-Comment[ja]=Dell I8K ハードウェアモニタ プラグイン
+Comment[ja]=Dell I8K ハードウェアモニタプラグイン
 Comment[kk]=Dell I8K жабдықтарын бақылау модулі
 Comment[km]=កម្មវិធី​ជំនួយ​របស់​កម្មវិធី​ត្រួតពិនិត្យផ្នែករឹង Dell I8K
 Comment[lt]=Dell I8K aparatinės įrangos stebėjimo priedas
Index: ksim/ksim.desktop
===================================================================
--- ksim/ksim.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ksim/ksim.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -107,6 +107,7 @@
 Comment[sr]=Надгледање системских информација (у стилу GKrellM-а)
 Comment[sr@Latn]=Nadgledanje sistemskih informacija (u stilu GKrellM-a)
 Comment[sv]=Systeminformationsövervakare (Gkrellm-stil)
+Comment[tr]=Sistem Bildirimileri İzleyici (GKrellM stili)
 Comment[uk]=Монітор системної інформації (в стилі GKrellM)
 Comment[zh_CN]=系统信息监视器(GKrellM 风格)
 Comment[zh_TW]=系統資訊監視器 (GKrellM 風格)
Index: kwallet/main.cpp
===================================================================
--- kwallet/main.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kwallet/main.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -94,7 +94,7 @@
 			wm.openWallet(args->arg(i));
 		}
 	}
-
+	args->clear();
 	return a.exec();
 }
 
Index: kwallet/kwalletmanager.desktop
===================================================================
--- kwallet/kwalletmanager.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kwallet/kwalletmanager.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -52,7 +52,7 @@
 Name[zh_CN]=钱包管理工具
 Name[zh_TW]=錢包管理工具
 Exec=kwalletmanager %u
-MimeType=application/x-kde-wallet
+MimeType=application/x-kde-wallet;
 InitialPreference=6
 Icon=kwalletmanager
 Type=Application
Index: kwallet/konfigurator/kwalletconfig.desktop
===================================================================
--- kwallet/konfigurator/kwalletconfig.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kwallet/konfigurator/kwalletconfig.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -51,7 +51,7 @@
 Name[zh_TW]=KDE 錢包
 Comment=KDE Wallet Configuration
 Comment[ar]=إعداد محفظة كدى
-Comment[bg]=Настройки на системата Портфейл
+Comment[bg]=Настройване на системата Портфейл
 Comment[br]=Kefluniadur Doug-paperoù KDE
 Comment[bs]=Podešavanje KDE novčanika
 Comment[ca]=Configuració de carteres KDE
Index: kdf/kdfconfig.h
===================================================================
--- kdf/kdfconfig.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdf/kdfconfig.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -50,7 +50,7 @@
       {
         mRes     = res;
         mName    = name;
-      };
+      }
       CTabName( void ) { }
       ~CTabName( void ) { }
 
Index: kdf/kwikdisk.desktop
===================================================================
--- kdf/kwikdisk.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdf/kwikdisk.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -75,3 +75,4 @@
 GenericName[zu]=Umsebenzi Wonozindaba Ogudluzekayo
 Terminal=false
 Categories=Qt;KDE;System;X-KDE-More;
+OnlyShowIn=KDE;
Index: kdf/kdfwidget.h
===================================================================
--- kdf/kdfwidget.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdf/kdfwidget.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -53,7 +53,7 @@
       mName    = name;
       mVisible = visible;
       mWidth   = width;
-    };
+    }
     CTabEntry( void ) { }
     ~CTabEntry( void ) { }
 
Index: kdf/disks.h
===================================================================
--- kdf/disks.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdf/disks.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -63,7 +63,7 @@
    *                 all this information is gained from the objects' data
    *                 if no mountCommand is set it defaults to "mount %d"
    **/
-  QString mountCommand() const { return mntcmd; };
+  QString mountCommand() const { return mntcmd; }
   /**
    * sets the used umountCommand for the actual DiskEntry.
    * @param mntcmd   is a string containing the executable file and
@@ -73,17 +73,17 @@
    *                 all this information is gained from the objects' data
    *                 if no umountCommand is set it defaults to "umount %d"
    **/
-  QString umountCommand() const { return umntcmd; };
-  QString fsType() const { return type; };
-  bool mounted() const { return isMounted; };
-  int kBSize() const { return size; };
+  QString umountCommand() const { return umntcmd; }
+  QString fsType() const { return type; }
+  bool mounted() const { return isMounted; }
+  int kBSize() const { return size; }
   QString iconName();
-  QString realIconName() { return icoName; };
-  QString prettyKBSize() const { return KIO::convertSizeFromKB(size); };
-  int kBUsed() const { return used; };
-  QString prettyKBUsed() const { return KIO::convertSizeFromKB(used); };
-  int kBAvail() const  { return avail; };
-  QString prettyKBAvail() const { return KIO::convertSizeFromKB(avail); };
+  QString realIconName() { return icoName; }
+  QString prettyKBSize() const { return KIO::convertSizeFromKB(size); }
+  int kBUsed() const { return used; }
+  QString prettyKBUsed() const { return KIO::convertSizeFromKB(used); }
+  int kBAvail() const  { return avail; }
+  QString prettyKBAvail() const { return KIO::convertSizeFromKB(avail); }
   float percentFull() const;
 
 signals:
Index: kdf/disklist.h
===================================================================
--- kdf/disklist.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdf/disklist.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -62,8 +62,8 @@
 class Disks : public DisksBase
 {
  public:
-  Disks(bool deepCopies=TRUE) { dc = deepCopies;};
-  ~Disks() { clear(); };
+  Disks(bool deepCopies=TRUE) { dc = deepCopies;}
+  ~Disks() { clear(); }
 private:
   int compareItems( DiskEntry s1, DiskEntry s2 )
   {
Index: kmilo/kmilod/kmilod.desktop
===================================================================
--- kmilo/kmilod/kmilod.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/kmilod/kmilod.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -53,6 +53,7 @@
 Comment[sr@Latn]=KDE-ov obavaštavač o posebnim tasterima
 Comment[sv]=Underrättelse om KDE specialtangent
 Comment[ta]= கேடிஇ சிறப்பு விசை குறிப்பான்
+Comment[tr]=KDE özel tuş bildirici
 Comment[uk]=Сповіщувач про натискання спеціальних клавіш для KDE
 Comment[zh_CN]=KDE 特殊按键通知程序
 Comment[zh_TW]=KDE 特殊鍵通知
Index: kmilo/thinkpad/kmilo_thinkpad.desktop
===================================================================
--- kmilo/thinkpad/kmilo_thinkpad.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/thinkpad/kmilo_thinkpad.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -12,7 +12,7 @@
 Name[cy]=Ategyn Thinkpad
 Name[de]=Thinkpad-Plugin
 Name[el]=Πρόσθετο Thinkpad
-Name[es]=Plugin para Thinkpad
+Name[es]=Complemento para Thinkpad
 Name[et]=Thinkpadi plugin
 Name[eu]=Thikpad Plugina
 Name[fa]=Thinkpad وصلۀ
Index: kmilo/thinkpad/kcmthinkpad/thinkpad.desktop
===================================================================
--- kmilo/thinkpad/kcmthinkpad/thinkpad.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/thinkpad/kcmthinkpad/thinkpad.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -56,7 +56,7 @@
 Name[zh_TW]=IBM Thinkpad 筆記型電腦
 Comment=Configure the KDE Interface to the IBM Thinkpad Special Controls
 Comment[ar]=إعداد واجهة كدى لمُتَحكّمات IBM Thinkpad الخاصة
-Comment[bg]=Настройки на IBM Thinkpad Special Controls
+Comment[bg]=Настройване на IBM Thinkpad
 Comment[bs]=Podesite KDE interfejs za IBM Thinkpad posebne kontrole
 Comment[ca]=Configura la interfície entre KDE i els controls especials de IBM Thinkpad
 Comment[cs]=Nastavení KDE rozhraní ovládání notebooků IBM Thinkpad
@@ -64,7 +64,7 @@
 Comment[da]=Indstil KDE's grænseflade til IBM Thinkpads specielle kontroller
 Comment[de]=KDE Schnittstelle zu den Sonderfunktionen von IBM Thinkpads einrichten
 Comment[el]=Ρυθμίστε το περιβάλλον χρήσης του KDE για τα ειδικά χειριστήρια του IBM Thinkpad
-Comment[es]=Configurar el interfaz de KDE para los controles especiales del IBM Thinkpad
+Comment[es]=Configurar la interfaz de KDE para los controles especiales del IBM Thinkpad
 Comment[et]=KDE liides IBM Thinkpadi eriklahvide seadistamiseks
 Comment[eu]=IBM Thinkpad-en Aginte Berezientzako KDE Interfazea Konfiguratu
 Comment[fa]=پیکربندی واسط KDEدر کلیدهای مهار ویژۀ  IBM Thinkpad
Index: kmilo/demo/kmilo_demo.desktop
===================================================================
--- kmilo/demo/kmilo_demo.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/demo/kmilo_demo.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -11,7 +11,7 @@
 Name[da]=Demo-plugin
 Name[de]=Demo-Plugin
 Name[el]=Πρόσθετο επίδειξης
-Name[es]=Plugin de pruebas
+Name[es]=Complemento de demostración
 Name[et]=Demoplugin
 Name[eu]=Demo Plugina
 Name[fa]=وصلۀ Demo
@@ -60,7 +60,7 @@
 Comment[da]=Dette plugin gør ingenting.
 Comment[de]=Dieses Plugin macht nichts.
 Comment[el]=Αυτό το πρόσθετο δεν κάνει τίποτα.
-Comment[es]=Este plugin no hace nada.
+Comment[es]=Este complemento no hace nada.
 Comment[et]=See plugin ei tee mitte kui midagi.
 Comment[eu]=Plugin honek ez du ezer egiten.
 Comment[fa]=این وصله کاری انجام نمی‌دهد.
Index: kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop
===================================================================
--- kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -57,7 +57,7 @@
 Name[zh_TW]=Sony Vaio 筆記型電腦
 Comment=Configure the KDE Interface to the Sony Programmable Interrupt Controller Driver
 Comment[ar]=إعداد واجهة كدى لجهاز لمشغل التحكّم بالمقاطعة القابل للبرمجة من سونى
-Comment[bg]=Настройки на Sony Programmable Interrupt Controller Driver
+Comment[bg]=Настройване на Sony Programmable Interrupt Controller Driver
 Comment[bs]=Podesite KDE interfejs za Sony Programmable Interrupt Controler drajver
 Comment[ca]=Configura la interfície entre KDE i el controlador d'interrupcions programables Sony
 Comment[cs]=Nastavení KDE rozhraní ovladače programovatelného řadiče přerušení Sony
@@ -65,7 +65,7 @@
 Comment[da]=Indstil KDE's grænseflade til Sony's programmerbare Interrupt Controller Driver
 Comment[de]=Einrichten der KDE-Schnittstelle zum "Sony Programmable Interrupt Controller" Treiber
 Comment[el]=Ρυθμίστε το περιβάλλον χρήσης του KDE για τον Sony Programmable Interrupt Controller Driver
-Comment[es]=Configurar el interfaz de KDE para el controlador programable de interrupciones de Sony
+Comment[es]=Configurar la interfaz de KDE para el controlador programable de interrupciones de Sony
 Comment[et]=KDE liides Sony programmeeritava katkestuskontrolleri draiveri seadistamiseks
 Comment[eu]=Konfiguratu Sonyren Interrupzio Kontrolatzaile Programagarriaren Gidariarentzako KDE Interfazea
 Comment[fa]=پیکربندی واسط KDE در گردانندۀ کنترل‌کنندۀ قابل برنامه‌ریزی دارای وقفه
Index: kmilo/kmilo_kvaio/kmilo_kvaio.desktop
===================================================================
--- kmilo/kmilo_kvaio/kmilo_kvaio.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/kmilo_kvaio/kmilo_kvaio.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -12,7 +12,7 @@
 Name[cy]=Ategyn Sony Vaio
 Name[da]=Sony Vaio plugin
 Name[el]=Πρόσθετο Sony Vaio
-Name[es]=Plugin para Sony Vaio
+Name[es]=Complemento para Sony Vaio
 Name[et]=Sony Vaio plugin
 Name[eu]=Sony Vaio Plugina
 Name[fa]=وصلۀ Sony Vaio
Index: kmilo/powerbook/kmilo_powerbook.desktop
===================================================================
--- kmilo/powerbook/kmilo_powerbook.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/powerbook/kmilo_powerbook.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -12,7 +12,7 @@
 Name[cy]=Ategyn Apple (R) PowerBook (R)
 Name[da]=Apple (R) PowerBook (R) plugin
 Name[el]=Πρόσθετο Apple (R) PowerBook (R)
-Name[es]=Plugin para Apple (R) PowerBook (R)
+Name[es]=Complemento para Apple (R) PowerBook (R)
 Name[et]=Apple (R) PowerBook (R) plugin
 Name[eu]=Apple (R) PowerBook (R) Plugina
 Name[fa]=Apple (R) PowerBook (R) وصلۀ
@@ -62,7 +62,7 @@
 Comment[da]=Dette plugin aktivere støtte for specielle taster på Apple (R) PowerBook (R) bærbare via pbbuttonsd.
 Comment[de]=Dieses Plugin schaltet die Unterstützung für Sondertasten des Apple (R) Powerbooks (R) via pbbuttonsd ein.
 Comment[el]=Αυτό το πρόσθετο ενεργοποιεί την υποστήριξη για ειδικά πλήκτρα στα φορητά Apple (R) PowerBook (R) μέσω του pbbuttonsd.
-Comment[es]=Este plugin activa el soporte de las teclas especiales en los portátiles Apple (R) PowerBook (R) mediante pbbuttonsd.
+Comment[es]=Este complemento activa el soporte de las teclas especiales en los portátiles Apple (R) PowerBook (R) mediante pbbuttonsd.
 Comment[et]=See plugin võimaldab pbbuttonsd vahendusel kasutada eriklahve Apple (R) PowerBook (R) sülearvutil.
 Comment[eu]=Plugin honek tekla berezientzako euskarria gaitzen du Apple (R) PowerBook (R) magalekoetan pbbuttonsd bitartez.
 Comment[fa]=این وصله، برای کلیدهای ویژۀ روی رایانه‌های کیفی  Apple (R) PowerBook (R)  via pbbuttonsd پشتیبانی را فعال می‌کند.
Index: kmilo/powerbook2/kmilo_powerbook.desktop
===================================================================
--- kmilo/powerbook2/kmilo_powerbook.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/powerbook2/kmilo_powerbook.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -11,7 +11,7 @@
 Name[cs]=Apple PowerBook modul
 Name[cy]=Ategyn Apple PowerBook
 Name[el]=Πρόσθετο Apple PowerBook
-Name[es]=Plugin del PowerBook de Apple
+Name[es]=Complemento del PowerBook de Apple
 Name[et]=Apple PowerBooki plugin
 Name[eu]=Apple PowerBook Plugina
 Name[fa]=Apple PowerBook وصلۀ
@@ -62,7 +62,7 @@
 Comment[da]=Dette plugin aktivere støtte for specielle taster på Apple PowerBooks via pbbuttonsd.
 Comment[de]=Dieses Plugin schaltet die Unterstützung für Sondertasten des Apple Powerbooks via pbbuttonsd ein.
 Comment[el]=Αυτό το πρόσθετο ενεργοποιεί την υποστήριξη για ειδικά πλήκτρα στα Apple PowerBooks μέσω του pbbuttonsd.
-Comment[es]=Este plugin activa el soporte de las teclas especiales en los PowerBooks de Apple mediante pbbuttonsd.
+Comment[es]=Este complemento activa el soporte de las teclas especiales en los PowerBooks de Apple mediante pbbuttonsd.
 Comment[et]=See plugin võimaldab pbbuttonsd vahendusel kasutada eriklahve Apple PowerBooksil
 Comment[eu]=Plugin honek tekla berezientzako euskarria gaitu du Apple PowerBook magalekoetan pbbuttonsd bitartez.
 Comment[fa]=این وصله، برای کلیدهای ویژۀ Apple PowerBooks via pbbuttonsd، پشتیبانی را فعال می‌کند.
Index: kmilo/asus/kmilo_asus.desktop
===================================================================
--- kmilo/asus/kmilo_asus.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/asus/kmilo_asus.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -12,7 +12,7 @@
 Name[da]=Asus bærbar plugin
 Name[de]=Asus Notebook-Plugin
 Name[el]=Πρόσθετο Sony Vaio
-Name[es]=Plugin para el portátil Asus
+Name[es]=Complemento para el portátil Asus
 Name[et]=Asuse sülearvuti plugin
 Name[eu]=Asus Laptop plugina
 Name[fa]=وصلۀ رایانۀ کیفی Asus
@@ -43,6 +43,7 @@
 Name[sr@Latn]=Priključak za Asus-ove laptope
 Name[sv]=Insticksprogram för Asus bärbar dator
 Name[ta]=Asus மடிக்கணினி சொருகுப்பொருள்
+Name[tr]=Asus Dizüstü Bilgisayar Eklentisi
 Name[uk]=Втулок лептопа Asus
 Name[uz]=Asus лаптопи учун плагин
 Name[zh_CN]=华硕笔记本插件
@@ -87,6 +88,7 @@
 Comment[sr@Latn]=Podrška za posebne tastere na Asus-ovim laptopima
 Comment[sv]=Aktiverar stöd för speciella tangenter på Asus bärbar dator
 Comment[ta]=விசேஷ ஆஸ்சஸ் மடிக்கணினி விசைகளுக்கான ஆதரவை செயல்படுத்துகிறது
+Comment[tr]=Asus dizüstü bilgisayarlar için özel tuş desteğini etkinleştirir
 Comment[uk]=Уможливлює підтримку спеціальних клавіш для лептопа Asus
 Comment[zh_CN]=启用华硕笔记本的特殊按键
 Comment[zh_TW]=開啟華碩筆記型電腦特殊鍵支援
Index: kmilo/delli8k/kmilo_delli8k.desktop
===================================================================
--- kmilo/delli8k/kmilo_delli8k.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/delli8k/kmilo_delli8k.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -12,7 +12,7 @@
 Name[da]=Dell I8k bærbar plugin
 Name[de]=Dell I8k Notebook-Plugin
 Name[el]=Πρόσθετο φορητού Dell I8k
-Name[es]=Plugin para el portátil Dell I8k
+Name[es]=Complemento para el portátil Dell I8k
 Name[et]=Dell I8k sülearvuti plugin
 Name[eu]=Dell I8k Laptop Plugina
 Name[fa]=Dell I8k وصلۀ رایانۀ کیفی
@@ -43,6 +43,7 @@
 Name[sr@Latn]=Priključak za Dell-ove I8k laptope
 Name[sv]=Insticksprogram för Dell I8k bärbar dator
 Name[ta]=டெல் 18கே மடிக்கணினி சொருகுப்பொருள்
+Name[tr]=Dell I8k Laptop Eklentisi
 Name[uk]=Втулок лептопа Dell I8k
 Name[uz]=Dell I8k лаптопи учун плагин
 Name[zh_CN]=Dell I8k 笔记本插件
@@ -86,6 +87,7 @@
 Comment[sr@Latn]=Podrška za posebne tastere na Dell-ovim laptopima
 Comment[sv]=Aktiverar stöd för speciella tangenter på Dell bärbar dator
 Comment[ta]=விசேஷ டெல் மடிக்கணினி விசைகளுக்கான ஆதரவை செயல்படுத்துகிறது
+Comment[tr]=Dell dizüstü bilgisayarların özel tuşları için desteği etkinleştirir
 Comment[uk]=Уможливлює підтримку спеціальних клавіш для лептопа Dell
 Comment[zh_CN]=此插件启用 Dell 笔记本的特殊按键
 Comment[zh_TW]=開啟 Dell 筆記型電腦特殊鍵支援
Index: kmilo/generic/kmilo_generic.desktop
===================================================================
--- kmilo/generic/kmilo_generic.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kmilo/generic/kmilo_generic.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -13,7 +13,7 @@
 Name[da]=Generisk tastatur-plugin
 Name[de]=Allgemeines Tastatur-Plugin
 Name[el]=Γενικευμένο πρόσθετο πληκτρολογίου
-Name[es]=Plugin de teclado genérico
+Name[es]=Complemento de teclado genérico
 Name[et]=Tavaline klaviatuuriplugin
 Name[eu]=Teklatu Arruntaren Plugina
 Name[fa]=وصلۀ کلی صفحه کلید
@@ -64,7 +64,7 @@
 Comment[da]=Dette plugin aktivere støtte for specielle taster på alle slags tastaturer og bærbare.
 Comment[de]=Dieses Plugin aktiviert die Unterstützung für die Sondertasten diverser Tastaturen und Notebooks.
 Comment[el]=Αυτό το πρόσθετο ενεργοποιεί την υποστήριξη για ειδικά πλήκτρα σε όλα τα είδη των πληκτρολογίων και φορητών υπολογιστών.
-Comment[es]=Este plugin activa el soporte de las teclas especiales en todo tipo de teclados y portátiles.
+Comment[es]=Este complemento activa el soporte de las teclas especiales en todo tipo de teclados y portátiles.
 Comment[et]=See plugin võimaldab kasutada eriklahve igasugustel klaviatuuridel ja sülearvutitel.
 Comment[eu]=Plugin honek tekla berezientzako euskarria gaitzen du teklatu eta magaleko mota guztietan.
 Comment[fa]=این وصله، برای کلیدهای ویژه روی همۀ انواع صفحه کلید و رایانه‌های کیفی، پشتیبانی را فعال می‌سازد.
Index: kregexpeditor/KWidgetStreamer/kwidgetstreamer.h
===================================================================
--- kregexpeditor/KWidgetStreamer/kwidgetstreamer.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kregexpeditor/KWidgetStreamer/kwidgetstreamer.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -51,7 +51,7 @@
   typedef QStringList::Iterator PropertyListIt;
 
   KWidgetStreamer();
-  virtual ~KWidgetStreamer() {};
+  virtual ~KWidgetStreamer() {}
 
   virtual void toStream(const QObject* from, QDataStream& stream );
   virtual void fromStream(QDataStream& stream, QObject* to);
Index: kregexpeditor/userdefinedregexps.h
===================================================================
--- kregexpeditor/userdefinedregexps.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kregexpeditor/userdefinedregexps.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -63,7 +63,7 @@
   RegExp* regExp() const;
   QString name() const;
   void setName( const QString& );
-  bool isUsersRegExp() const { return _usersRegExp; };
+  bool isUsersRegExp() const { return _usersRegExp; }
 
 private:
   QString _name;
Index: kregexpeditor/regexp.h
===================================================================
--- kregexpeditor/regexp.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kregexpeditor/regexp.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -50,7 +50,7 @@
 
     enum RegExpType { CONC, TEXT, DOT, POSITION, REPEAT, ALTN, COMPOUND, LOOKAHEAD, TEXTRANGE };
     virtual RegExpType type() const = 0;
-    virtual void replacePart( CompoundRegExp* /* replacement */ ) {};
+    virtual void replacePart( CompoundRegExp* /* replacement */ ) {}
     bool isSelected() const { return _selected; }
     void setSelected( bool b ) { _selected = b; }
 
@@ -58,7 +58,7 @@
     RegExp* readRegExp( QDomElement top, const QString& version );
 
 private:
-    RegExp() {}; // disable
+    RegExp() {} // disable
     QPtrList<RegExp> _children;
     RegExp* _parent;
     bool _destructing;
Index: kregexpeditor/pair.h
===================================================================
--- kregexpeditor/pair.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kregexpeditor/pair.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -24,7 +24,7 @@
 template<class F, class S> class Pair
 {
 public:
-  Pair() {};
+  Pair() {}
 	Pair(F first, S second) : _first(first), _second(second) {}
 	F first() { return _first; }
 	S second() { return _second; }
Index: kregexpeditor/KMultiFormListBox/kmultiformlistboxentry.h
===================================================================
--- kregexpeditor/KMultiFormListBox/kmultiformlistboxentry.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kregexpeditor/KMultiFormListBox/kmultiformlistboxentry.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -59,7 +59,7 @@
 
   // This function must return a string representing the KMultiFormListBox. This is
   // used when showing the fast-search menu available from the `Idx' button.
-  virtual QString idxString() { return QString::fromLatin1(""); };
+  virtual QString idxString() { return QString::fromLatin1(""); }
 
 public slots:
   void acceptIndexButton();
Index: kdelirc/kcmlirc/kcmlirc.desktop
===================================================================
--- kdelirc/kcmlirc/kcmlirc.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdelirc/kcmlirc/kcmlirc.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -3,7 +3,7 @@
 Icon=irkick
 Comment=Configure your remote controls for use with applications
 Comment[ar]=إعداد التحكّم عن بعد لاستخدامه في تطبيقاتك
-Comment[bg]=Настройки на дистанционно управление
+Comment[bg]=Настройване на дистанционно управление
 Comment[bs]=Podesite vaš daljinski upravljač za upotrebu sa programima
 Comment[ca]=Configura els teus controls remots per usar amb aplicacions
 Comment[cs]=Nastavení dálkových ovladačů
Index: kdelirc/irkick/irkick.desktop
===================================================================
--- kdelirc/irkick/irkick.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdelirc/irkick/irkick.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -44,6 +44,7 @@
 GenericName[sr@Latn]=KDE-ov LIRC server
 GenericName[sv]=KDE:s LIRC-server
 GenericName[ta]=KDE LIRC சேவகன்
+GenericName[tr]=KDE LIRC Sunucusu
 GenericName[uk]=Сервер LIRC для KDE
 GenericName[uz]=KDE LIRC сервери
 GenericName[zh_CN]=KDE LIRC 服务器
@@ -53,8 +54,8 @@
 X-KDE-autostart-after=panel
 X-KDE-StartupNotify=false
 X-DCOP-ServiceType=Unique
+X-KDE-autostart-condition=irkickrc:General:AutoStart:false
 Categories=Qt;KDE;Utility;X-KDE-Utilities-Peripherals;
-X-KDE-autostart-condition=irkickrc:General:AutoStart:false
 
 Exec=irkick
 Icon=irkick
@@ -109,4 +110,3 @@
 Comment[zh_CN]=KDE Linux 红外远程控制服务器。
 Comment[zh_TW]=KDE Linux 紅外線遙控伺服器。
 Encoding=UTF-8
-
Index: kdelirc/kdelirc/iraction.h
===================================================================
--- kdelirc/kdelirc/iraction.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kdelirc/kdelirc/iraction.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -82,7 +82,7 @@
 	void setUnique(const bool a) { theUnique = a; }
 
 	IRAction(const QString &newProgram, const QString &newObject, const QString &newMethod, const Arguments &newArguments, const QString &newRemote, const QString &newMode, const QString &newButton, const bool newRepeat, const bool newAutoStart, const bool newDoBefore, const bool newDoAfter, const bool newUnique, const IfMulti newIfMulti);
-	IRAction() { theProgram = QString::null; };
+	IRAction() { theProgram = QString::null; }
 };
 
 #endif
Index: kcalc/kcalc.desktop
===================================================================
--- kcalc/kcalc.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ kcalc/kcalc.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -13,7 +13,6 @@
 Name[ta]=கேகணிப்பான்
 Name[tg]=KМошинаи ҳисобкунак
 Name[th]=เครื่องคิดเลข
-Name[tr]=Hesap Makinesi
 Name[uz]=Калкулятор
 Name[zh_TW]=KDE 計算機
 Exec=kcalc  -caption "%c" %i %m
Index: ark/arch.h
===================================================================
--- ark/arch.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ark/arch.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -114,7 +114,7 @@
     void unarchFile( QStringList *, const QString & _destDir,
                              bool viewFriendly = false );
 
-    QString fileName() const { return m_filename; };
+    QString fileName() const { return m_filename; }
 
     enum EditProperties{ Add = 1, Delete = 2, Extract = 4,
                          View = 8, Integrity = 16 };
Index: ark/ark.desktop
===================================================================
--- ark/ark.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ ark/ark.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Encoding=UTF-8
-MimeType=application/x-gzip;application/x-lha;application/x-tar;application/x-tgz;application/x-tbz;application/x-tbz2;application/x-zip;application/x-bzip;application/x-tzo;application/x-lzop;application/x-rar;application/x-zoo;application/x-tarz;application/x-archive;application/x-bzip2;application/x-jar;application/x-deb;application/x-ace;application/x-7z;application/x-arc;application/x-arj;application/x-compress;application/x-cpio;application/x-pak
+MimeType=application/x-gzip;application/x-lha;application/x-tar;application/x-tgz;application/x-tbz;application/x-tbz2;application/x-zip;application/x-bzip;application/x-tzo;application/x-lzop;application/x-rar;application/x-zoo;application/x-tarz;application/x-archive;application/x-bzip2;application/x-jar;application/x-deb;application/x-ace;application/x-7z;application/x-arc;application/x-arj;application/x-compress;application/x-cpio;application/x-pak;
 GenericName=Archiving Tool
 GenericName[af]=Argiveer Program
 GenericName[ar]=أداة أرشفة
Index: superkaramba/src/memsensor.cpp
===================================================================
--- superkaramba/src/memsensor.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ superkaramba/src/memsensor.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -32,6 +32,8 @@
 #include <sys/swap.h>
 #endif
 
+#include <kprocio.h>
+
 #if defined Q_OS_FREEBSD || defined(Q_OS_NETBSD)
 /* define pagetok in terms of pageshift */
 #define pagetok(size) ((size) << pageshift)
@@ -51,7 +53,9 @@
 
     /* we only need the amount of log(2)1024 for our conversion */
     pageshift -= 10;
-# if (defined(Q_OS_FREEBSD) && __FreeBSD_version < 500018)
+# if defined(Q_OS_FREEBSD) && defined(__FreeBSD_version) && __FreeBSD_version >= 500018
+    kd = kvm_open("/dev/null", "/dev/null", "/dev/null", O_RDONLY, "kvm_open");
+# elif defined Q_OS_FREEBSD
     connect(&ksp, SIGNAL(receivedStdout(KProcess *, char *, int )),
             this,SLOT(receivedStdout(KProcess *, char *, int )));
     connect(&ksp, SIGNAL(processExited(KProcess *)),
@@ -62,8 +66,6 @@
     MaxSet = false;
 
     readValues();
-# elif defined Q_OS_FREEBSD
-    kd = kvm_open("/dev/null", "/dev/null", "/dev/null", O_RDONLY, "kvm_open");
 # endif
 #else
     readValues();
@@ -180,9 +182,7 @@
 int MemSensor::getSwapTotal()
 {
 #ifdef Q_OS_FREEBSD
-# if __FreeBSD_version < 500018
-    return(swapTotal);
-# else
+# if defined(__FreeBSD_version) && __FreeBSD_version >= 500018
     int n = -1;
     int pagesize = getpagesize();
     int retavail = 0;
@@ -196,6 +196,8 @@
     retavail = swapinfo.ksw_total * pagesize / 1024;
 
     return(retavail);
+#else
+    return(swapTotal);
 # endif
 #elif defined(Q_OS_NETBSD)
     struct uvmexp_sysctl uvmexp;
@@ -222,9 +224,7 @@
 int MemSensor::getSwapFree()
 {
 #ifdef Q_OS_FREEBSD
-# if __FreeBSD_version < 500018
-    return(swapTotal - swapUsed);
-# else
+# if defined(__FreeBSD_version) && __FreeBSD_version >= 500018
     int n = -1;
     int pagesize = getpagesize();
     int retfree = 0;
@@ -237,6 +237,8 @@
     retfree = (swapinfo.ksw_total - swapinfo.ksw_used) * pagesize / 1024;
 
     return(retfree);
+# else
+    return(swapTotal - swapUsed);
 # endif
 #elif defined(Q_OS_NETBSD)
     struct uvmexp_sysctl uvmexp;
@@ -267,7 +269,7 @@
 void MemSensor::readValues()
 {
 #if defined Q_OS_FREEBSD || defined(Q_OS_NETBSD)
-# if (defined(Q_OS_FREEBSD) && __FreeBSD_version < 500018)
+# if defined(Q_OS_FREEBSD) && !(defined(__FreeBSD_version) && __FreeBSD_version >= 500018)
     ksp.clearArguments();
     ksp << "swapinfo";
     ksp.start( KProcess::NotifyOnExit,KProcIO::Stdout);
@@ -291,7 +293,7 @@
     SensorParams *sp;
     Meter *meter;
     QObjectListIt it( *objList );
-#if (defined(Q_OS_FREEBSD) && __FreeBSD_version < 500018)
+#if defined(Q_OS_FREEBSD) && !(defined(__FreeBSD_version) && __FreeBSD_version >= 500018)
     bool set = false;
 #endif
     int totalMem = getMemTotal();
@@ -303,7 +305,7 @@
     while (it != 0)
     {
         sp = (SensorParams*)(*it);
-#if (defined(Q_OS_FREEBSD) && __FreeBSD_version < 500018)
+#if defined(Q_OS_FREEBSD) && !(defined(__FreeBSD_version) && __FreeBSD_version >= 500018)
         if ( (!MaxSet) && (totalSwap > 0) ) {
            setMaxValue(sp);
            bool set = true;
@@ -331,7 +333,7 @@
         meter->setValue(format);
         ++it;
     }
-#if (defined(Q_OS_FREEBSD) && __FreeBSD_version < 500018)
+#if defined(Q_OS_FREEBSD) && !(defined(__FreeBSD_version) && __FreeBSD_version >= 500018)
     if (set)
         MaxSet = true;
 #endif
Index: superkaramba/src/textlabel.h
===================================================================
--- superkaramba/src/textlabel.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ superkaramba/src/textlabel.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -24,8 +24,8 @@
 {
 Q_OBJECT
 public:
-    typedef enum ScrollType { ScrollNone, ScrollNormal,
-                              ScrollBackAndForth, ScrollOnePass };
+    enum ScrollType { ScrollNone, ScrollNormal,
+                      ScrollBackAndForth, ScrollOnePass };
 
     TextLabel(karamba *k, int x,int y,int w,int h);
     TextLabel(karamba *k);
Index: superkaramba/src/misc_python.cpp
===================================================================
--- superkaramba/src/misc_python.cpp	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ superkaramba/src/misc_python.cpp	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -653,7 +653,7 @@
 #include <sys/ioctl.h>
 #include <net/if.h>
 #include <arpa/inet.h>
-#ifdef __FreeBSD__
+#if defined(__FreeBSD__) || defined(__DragonFly__)
 #include <netinet/in.h>
 #endif
 #if defined(Q_OS_SOLARIS)
Index: superkaramba/src/memsensor.h
===================================================================
--- superkaramba/src/memsensor.h	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ superkaramba/src/memsensor.h	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -44,18 +44,17 @@
 private:
   QString meminfo;
   void readValues();
-#if defined __FreeBSD__ || defined(Q_OS_NETBSD)
+#if defined(Q_OS_FREEBSD) || defined(Q_OS_NETBSD)
   int pageshift;           /* log base 2 of the pagesize */
   QString sensorResult;
   int swapTotal;
   int swapUsed;
-# if (defined(__FreeBSD__) && __FreeBSD_version < 500018)
-  KShellProcess ksp;
-  bool MaxSet;
-
-# elif defined __FreeBSD__
+# if defined(Q_OS_FREEBSD) && defined(__FreeBSD_version) && __FreeBSD_version >= 500018
   kvm_t *kd;
   kvm_swap swapinfo;
+# elif defined(Q_OS_FREEBSD)
+  KShellProcess ksp;
+  bool MaxSet;
 # endif
 #endif
 
Index: superkaramba/src/superkaramba.desktop
===================================================================
--- superkaramba/src/superkaramba.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ superkaramba/src/superkaramba.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -8,7 +8,7 @@
 Exec=superkaramba %U
 Icon=superkaramba
 Type=Application
-MimeType=application/x-superkaramba
+MimeType=application/x-superkaramba;
 Comment=An engine for cool desktop eyecandy.
 Comment[ar]=محرك لجماليات سطح مكتب لطيفة.
 Comment[bg]=Модул за разкрасяване на работния плот
@@ -46,6 +46,7 @@
 Comment[sr]=Машина за лицкање радне површине.
 Comment[sr@Latn]=Mašina za lickanje radne površine.
 Comment[sv]=Ett gränssnitt för häftigt ögongodis på skrivbordet.
+Comment[tr]=Masaüstünde güzel görünüm sağlayan bir motor.
 Comment[uk]=Рушій для елементів стільниці.
 Comment[zh_CN]=桌面养眼引擎。
 Comment[zh_TW]=桌面佈景選擇程式
@@ -88,6 +89,7 @@
 GenericName[sr]=Додаци десктопа
 GenericName[sr@Latn]=Dodaci desktopa
 GenericName[sv]=Grafiska skrivbordskomponenter
+GenericName[tr]=Masaüstü Ögeleri
 GenericName[uk]=Віджети стільниці
 GenericName[zh_CN]=桌面部件
 GenericName[zh_TW]=桌面元件
Index: superkaramba/mimetypes/x-superkaramba.desktop
===================================================================
--- superkaramba/mimetypes/x-superkaramba.desktop	(.../tags/KDE/3.5.7/kdeutils)	(wersja 679692)
+++ superkaramba/mimetypes/x-superkaramba.desktop	(.../branches/KDE/3.5/kdeutils)	(wersja 679692)
@@ -39,6 +39,7 @@
 Comment[sr]=Фајл теме СуперКарамбе
 Comment[sr@Latn]=Fajl teme SuperKarambe
 Comment[sv]=Superkaramba-temafil
+Comment[tr]=Superkaramba Tema Dosyası
 Comment[uk]=Файл теми Superkaramba
 Comment[zh_CN]=Superkaramba 主题文件
 Comment[zh_TW]=Superkaramba 佈景檔案

Zmiany atrybutów dla: .
___________________________________________________________________
Nazwa: svn:externals
   + admin https://svn.kde.org/home/kde/branches/KDE/3.5/kde-common/admin


