--- kdeutils-3.0.3/klaptopdaemon/power.cpp--	2002-08-26 16:03:54.000000000 +0200
+++ kdeutils-3.0.3/klaptopdaemon/power.cpp	2002-08-26 16:04:06.000000000 +0200
@@ -151,6 +151,10 @@ PowerConfig::PowerConfig (QWidget * pare
   load();
 }
 
+PowerConfig::~PowerConfig()
+{
+  delete config;
+}
 
 void PowerConfig::save()
 {
--- kdeutils-3.0.3/klaptopdaemon/power.h--	2002-08-26 16:17:08.000000000 +0200
+++ kdeutils-3.0.3/klaptopdaemon/power.h	2002-08-26 16:19:46.000000000 +0200
@@ -39,7 +39,7 @@ class PowerConfig : public KCModule
   Q_OBJECT
 public:
   PowerConfig( QWidget *parent=0, const char* name=0);
-
+  ~PowerConfig();
   void save( void );
   void load();
   void defaults();
