--- kdeutils-3.0.3/kdf/optiondialog.cpp--	2002-08-28 11:35:57.000000000 +0200
+++ kdeutils-3.0.3/kdf/optiondialog.cpp	2002-08-28 11:36:34.000000000 +0200
@@ -43,6 +43,9 @@ COptionDialog::COptionDialog( QWidget *p
 
 COptionDialog::~COptionDialog( void )
 {
+    delete mConf;
+    delete mMnt;
+
 }
 
 
--- kdeutils-3.0.3/kdf/kdfconfig.cpp--	2002-08-28 11:34:32.000000000 +0200
+++ kdeutils-3.0.3/kdf/kdfconfig.cpp	2002-08-28 11:35:00.000000000 +0200
@@ -145,6 +145,9 @@ KDFConfigWidget::KDFConfigWidget(QWidget
 
 KDFConfigWidget::~KDFConfigWidget()
 {
+		for ( int i = 0;i<8;i++)
+			delete mTabName[i];
+		
 }
 
 
--- kdeutils-3.0.3/kdf/disklist.cpp--	2002-08-28 11:33:07.000000000 +0200
+++ kdeutils-3.0.3/kdf/disklist.cpp	2002-08-28 11:33:36.000000000 +0200
@@ -91,6 +91,7 @@ if (NO_FS_TYPE)
 **/
 DiskList::~DiskList()
 {
+		delete dfProc;
 };
 
 
