--- kdeutils-3.0.3/klaptopdaemon/warning.h--	2002-08-26 16:00:10.000000000 +0200
+++ kdeutils-3.0.3/klaptopdaemon/warning.h	2002-08-26 16:00:32.000000000 +0200
@@ -40,7 +40,7 @@ class WarningConfig : public KCModule
   Q_OBJECT
 public:
   WarningConfig(int x, QWidget *parent=0, const char* name=0);
-
+  ~WarningConfig();
   void save( void );
   void load();
   void defaults();
--- kdeutils-3.0.3/klaptopdaemon/warning.cpp--	2002-08-26 15:59:40.000000000 +0200
+++ kdeutils-3.0.3/klaptopdaemon/warning.cpp	2002-08-26 16:01:17.000000000 +0200
@@ -172,6 +172,10 @@ WarningConfig::WarningConfig (int t, QWi
   my_load(1);
 }
 
+WarningConfig::~WarningConfig()
+{
+  delete config;
+}
 
 void WarningConfig::save()
 {
