diff -urN -x CVS kdeutils.orig/kdelirc/irkick/irkick.desktop kdeutils/kdelirc/irkick/irkick.desktop
--- kdeutils.orig/kdelirc/irkick/irkick.desktop	Fri Oct  1 15:54:46 2004
+++ kdeutils/kdelirc/irkick/irkick.desktop	Mon Oct  4 15:26:55 2004
@@ -26,7 +26,7 @@
 Name[sk]=KDE LIRC server
 Name[sl]=Strežnik KDE LIRC
 Name[sr]=KDE-ов LIRC сервер
-Name[sr@Latn]=KDE-ов LIRC сервер
+Name[sr@Latn]=KDE-ov LIRC server
 Name[sv]=KDE:s LIRC-server
 Name[ta]= கேடிஇ LIRC சேவகன்
 Name[tg]=KDE LIRC Хидматрасон
diff -urN -x CVS kdeutils.orig/klaptopdaemon/applnk/klaptopdaemon.desktop kdeutils/klaptopdaemon/applnk/klaptopdaemon.desktop
--- kdeutils.orig/klaptopdaemon/applnk/klaptopdaemon.desktop	Fri Oct  1 15:54:48 2004
+++ kdeutils/klaptopdaemon/applnk/klaptopdaemon.desktop	Mon Oct  4 15:26:57 2004
@@ -36,7 +36,7 @@
 Comment[sk]=Monitor nabitia bateriek
 Comment[sl]=Nadzornik baterije za prenosnik
 Comment[sr]=Монитор батерија лаптопа
-Comment[sr@Latn]=Монитор батерија лаптопа
+Comment[sr@Latn]=Monitor baterija laptopa
 Comment[sv]=Batterimonitor för bärbar dator
 Comment[ta]= மடிக் கணினி மின்கலம் கண்காணிப்பாளன்
 Comment[tg]=Дидабони батареяи компютери дастӣ
diff -urN -x CVS kdeutils.orig/klaptopdaemon/daemondock.cpp kdeutils/klaptopdaemon/daemondock.cpp
--- kdeutils.orig/klaptopdaemon/daemondock.cpp	Mon Aug  2 12:28:48 2004
+++ kdeutils/klaptopdaemon/daemondock.cpp	Mon Oct  4 13:31:04 2004
@@ -4,7 +4,7 @@
  *
  * This file contains the docked widget for the laptop battery monitor
  *
- * $Id$
+ * $Id$
  *
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
@@ -497,11 +497,7 @@
 {
   DCOPClient* client = kapp->dcopClient();
   if (client)
-    {
-      client->attach();
       client->send("kdesktop", "KScreensaverIface", "lock()", "");
-      client->detach();
-    }
   laptop_portable::invoke_hibernation();
 }
 void laptop_dock::invokeStandby()
@@ -518,11 +514,7 @@
 {
   DCOPClient* client = kapp->dcopClient();
   if (client)
-    {
-      client->attach();
       client->send("kdesktop", "KScreensaverIface", "lock()", "");
-      client->detach();
-    }
   laptop_portable::invoke_suspend();
 }
 
diff -urN -x CVS kdeutils.orig/kmilo/demo/kmilo_demo.desktop kdeutils/kmilo/demo/kmilo_demo.desktop
--- kdeutils.orig/kmilo/demo/kmilo_demo.desktop	Wed Sep  1 16:19:09 2004
+++ kdeutils/kmilo/demo/kmilo_demo.desktop	Mon Oct  4 15:27:00 2004
@@ -30,7 +30,7 @@
 Name[sk]=Testovací modul
 Name[sl]=Demonstracijski vstavek
 Name[sr]=Демонстрациони прикључак
-Name[sr@Latn]=Demonstracioni dodatak
+Name[sr@Latn]=Demonstracioni priključak
 Name[sv]=Demoinsticksprogram
 Name[ta]= முன்னோட்ட சொருகி
 Name[tg]=Модули Намоиш
@@ -68,7 +68,7 @@
 Comment[sk]=Tento modul nič nerobí.
 Comment[sl]=Ta vstavek ne naredi ničesar.
 Comment[sr]=Овај прикључак ништа не ради.
-Comment[sr@Latn]=Ovaj dodatak ništa ne radi.
+Comment[sr@Latn]=Ovaj priključak ništa ne radi.
 Comment[sv]=Det här insticksprogrammet gör ingenting.
 Comment[ta]= செருகுப்பொருள் செயல்படவில்லை.
 Comment[tg]=Ин модул ҳеҷ чиз намекунад.
diff -urN -x CVS kdeutils.orig/kmilo/generic/kmilo_generic.desktop kdeutils/kmilo/generic/kmilo_generic.desktop
--- kdeutils.orig/kmilo/generic/kmilo_generic.desktop	Wed Sep  1 16:19:09 2004
+++ kdeutils/kmilo/generic/kmilo_generic.desktop	Mon Oct  4 15:27:00 2004
@@ -32,7 +32,7 @@
 Name[sk]=Modul pre generickú klávesnicu
 Name[sl]=Generični vstavek za tipkovnice
 Name[sr]=Генерички прикључак тастатуре
-Name[sr@Latn]=Generički dodatak tastature
+Name[sr@Latn]=Generički priključak tastature
 Name[sv]=Allmänt insticksprogram för tangentbord
 Name[ta]= பொதுவான விசைப்பலகை சொருகுபொருள் 
 Name[tg]=Модули Умумии Забонак
@@ -70,7 +70,7 @@
 Comment[sk]=Tento modul obsahuje podporu pre špeciálne klávesy na rôznych klávesniciach a prenosných počítačoch.
 Comment[sl]=Ta vstavek omogoča podporo za posebne tipke na vseh vrstah tipkovnic in prenosnikov.
 Comment[sr]=Овај прикључак омогућава подршку за специјалне тастере на свакаквим тастатурама и лаптоповима.
-Comment[sr@Latn]=Ovaj dodatak omogućava podršku za specijalne tastere na svakakvim tastaturama i laptopovima.
+Comment[sr@Latn]=Ovaj priključak omogućava podršku za specijalne tastere na svakakvim tastaturama i laptopovima.
 Comment[sv]=Det här insticksprogrammet aktiverar stöd för speciella tangenter för alla typer av tangentbord och bärbara datorer.
 Comment[ta]=எல்லா அடுக்குகளிலும் உள்ள விசைபலகைகள் மற்றும் மடிக் கணினிகளின் விசேஷ விசைகளுக்கு இந்த சொருகு சாதனம் ஆதரவளிக்கிறது.
 Comment[tg]=Ин модул пуштибонии калидҳои махсусро дар ҳамаи навъҳои забонакҳо ва компютерҳои дастӣ дармегиронад.
diff -urN -x CVS kdeutils.orig/kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop kdeutils/kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop
--- kdeutils.orig/kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop	Fri Oct  1 15:54:50 2004
+++ kdeutils/kmilo/kmilo_kvaio/kcmkvaio/kvaio.desktop	Mon Oct  4 15:27:00 2004
@@ -29,7 +29,7 @@
 Name[sk]=Sony Vaio
 Name[sl]=Prenosnik Sony Vaio
 Name[sr]=Прикључак за Sony-јев Vaio лаптоп
-Name[sr@Latn]=Прикључак за Sony-јев Vaio лаптоп
+Name[sr@Latn]=Priključak za Sony-jev Vaio laptop
 Name[sv]=Sony Vaio bärbar dator
 Name[ta]=சோனி வையோ மடிக் கணிப்பொறி
 Name[tg]=Компютери Дастии Sony Vaio
diff -urN -x CVS kdeutils.orig/kmilo/kmilo_kvaio/kmilo_kvaio.desktop kdeutils/kmilo/kmilo_kvaio/kmilo_kvaio.desktop
--- kdeutils.orig/kmilo/kmilo_kvaio/kmilo_kvaio.desktop	Wed Sep  1 16:19:09 2004
+++ kdeutils/kmilo/kmilo_kvaio/kmilo_kvaio.desktop	Mon Oct  4 15:27:00 2004
@@ -31,7 +31,7 @@
 Name[sk]=Modul pre Sony Vaio
 Name[sl]=Vstavek za Sony Vaio
 Name[sr]=Прикључак за Sony-јев Vaio
-Name[sr@Latn]=Dodatak za Sony-jev Vaio
+Name[sr@Latn]=Priključak za Sony-jev Vaio
 Name[sv]=Sony Vaio-insticksprogram
 Name[ta]=சோனி வையோ சொருகு சாதனம்
 Name[tg]=Модули Sony Vaio
diff -urN -x CVS kdeutils.orig/kmilo/kmilod/kmilod.desktop kdeutils/kmilo/kmilod/kmilod.desktop
--- kdeutils.orig/kmilo/kmilod/kmilod.desktop	Fri Oct  1 15:54:50 2004
+++ kdeutils/kmilo/kmilod/kmilod.desktop	Mon Oct  4 15:27:01 2004
@@ -35,7 +35,7 @@
 Comment[sk]=KDE upozornenie na špeciálne klávesy
 Comment[sl]=Opomnik posebnih ključev v KDE
 Comment[sr]=KDE-ов обаваштавач о посебним тастерима
-Comment[sr@Latn]=KDE-ов обаваштавач о посебним тастерима
+Comment[sr@Latn]=KDE-ov obavaštavač o posebnim tasterima
 Comment[sv]=Underrättelse om KDE specialtangent
 Comment[ta]= கேடிஇ சிறப்பு விசை குறிப்பான்
 Comment[tg]=Огоҳкунандаи калиди махсуси KDE
diff -urN -x CVS kdeutils.orig/kmilo/powerbook/kmilo_powerbook.desktop kdeutils/kmilo/powerbook/kmilo_powerbook.desktop
--- kdeutils.orig/kmilo/powerbook/kmilo_powerbook.desktop	Wed Sep  1 16:19:10 2004
+++ kdeutils/kmilo/powerbook/kmilo_powerbook.desktop	Mon Oct  4 15:27:01 2004
@@ -31,7 +31,7 @@
 Name[sk]=Modul pre Apple (R) PowerBook (R)
 Name[sl]=Vstavek za Apple (R) PowerBook (R)
 Name[sr]=Прикључак за Apple-ов PowerBook
-Name[sr@Latn]=Dodatak za Apple-ov PowerBook
+Name[sr@Latn]=Priključak za Apple-ov PowerBook
 Name[sv]=Apple ® PowerBook ®-insticksprogram
 Name[ta]=ஆப்பிள் (R) பவர்புக் (R) சொருகுப்பொருள்
 Name[tg]=Модули Apple (R) PowerBook (R)
@@ -68,7 +68,7 @@
 Comment[sk]=Tento modul obsahuje podporu špeciálnych kláves pre Apple (R) PowerBook (R) pomocou pbbuttonsd.
 Comment[sl]=Ta vstavek vam omogoča podboro posebnim ključem na prenosnikih Apple (R) PowerBook (R) preko pbbuttonsd.
 Comment[sr]=Овај прикључак омогућава подршку за специјалне тастере на Apple-овом PowerBook лаптопу преко pbbuttonsd.
-Comment[sr@Latn]=Ovaj dodatak omogućava podršku za specijalne tastere na Apple-ovom PowerBook laptopu preko pbbuttonsd.
+Comment[sr@Latn]=Ovaj priključak omogućava podršku za specijalne tastere na Apple-ovom PowerBook laptopu preko pbbuttonsd.
 Comment[sv]=Det här insticksprogrammet aktiverar stöd för speciella tangenter för Apple ® PowerBook ® via pbbuttonsd.
 Comment[ta]=ஆப்பிள் (R) பவர்புக் (R) மடிக் கணினி மூலமாக பிபிபட்டன்எஸ்டி போன்றவைக்கு இந்த சொருகு பொருள் ஆதரிக்கிறது.
 Comment[tg]=Ин модул пуштибонии калидҳои махсусро дар компютери дастии Apple (R) PowerBook (R) ба воситаи pbbuttonsd дармегиронад.
diff -urN -x CVS kdeutils.orig/kmilo/powerbook2/kmilo_powerbook.desktop kdeutils/kmilo/powerbook2/kmilo_powerbook.desktop
--- kdeutils.orig/kmilo/powerbook2/kmilo_powerbook.desktop	Wed Sep  1 16:19:11 2004
+++ kdeutils/kmilo/powerbook2/kmilo_powerbook.desktop	Mon Oct  4 15:27:01 2004
@@ -31,7 +31,7 @@
 Name[sk]=Module pre Applet PowerBook
 Name[sl]=Vstavek za Apple PowerBook
 Name[sr]=Прикључак за Apple-ов PowerBook
-Name[sr@Latn]=Dodatak za Apple-ov PowerBook
+Name[sr@Latn]=Priključak za Apple-ov PowerBook
 Name[sv]=Apple PowerBook-insticksprogram
 Name[ta]=ஆப்பிள் பவர்புக் சொருகுப்பொருள்
 Name[tg]=Модули Apple PowerBook
@@ -69,7 +69,7 @@
 Comment[sk]=Tento modul obsahuje podporu špeciálnych kláves pre Apple (R) PowerBook (R) pomocou pbbuttonsd.
 Comment[sl]=Ta vstavek vam omogoča podboro posebnim ključem na prenosnikih Apple PowerBook preko pbbuttonsd.
 Comment[sr]=Овај прикључак омогућава подршку за специјалне тастере на Apple-овим PowerBook-овима преко pbbuttonsd.
-Comment[sr@Latn]=Ovaj dodatak omogućava podršku za specijalne tastere na Apple-ovim PowerBook-ovima preko pbbuttonsd.
+Comment[sr@Latn]=Ovaj priključak omogućava podršku za specijalne tastere na Apple-ovim PowerBook-ovima preko pbbuttonsd.
 Comment[sv]=Det här insticksprogrammet aktiverar stöd för speciella tangenter för Apple PowerBook via pbbuttonsd.
 Comment[ta]=ஆப்பிள் பவர்புக்ஸ் மூலமாக பிபிபட்டன்எஸ்டிக்கு இந்த சொருகுப் பொருள் ஆதரவளிக்கிறது
 Comment[tg]=Ин модул пуштибонии калидҳои махсусро дар компютери дастии Apple PowerBooks ба воситаи pbbuttonsd дармегиронад.
diff -urN -x CVS kdeutils.orig/kmilo/thinkpad/kcmthinkpad/thinkpad.desktop kdeutils/kmilo/thinkpad/kcmthinkpad/thinkpad.desktop
--- kdeutils.orig/kmilo/thinkpad/kcmthinkpad/thinkpad.desktop	Fri Oct  1 15:54:51 2004
+++ kdeutils/kmilo/thinkpad/kcmthinkpad/thinkpad.desktop	Mon Oct  4 15:27:02 2004
@@ -27,7 +27,7 @@
 Name[sk]=IBM Thinkpad
 Name[sl]=Prenosnik IBM Thinkpad
 Name[sr]=IBM Thinkpad лаптоп
-Name[sr@Latn]=IBM Thinkpad лаптоп
+Name[sr@Latn]=IBM Thinkpad laptop
 Name[sv]=IBM Thinkpad bärbar dator
 Name[ta]=ஐபிஎம் திங்க்பாட் மடிக்கணினி
 Name[tg]=Компютери Дастии IBM Thinkpad
@@ -59,7 +59,7 @@
 Comment[sk]=Nastavenie rozhrania KDE k špeciálnym ovládacím prvkom notebooku IBM Thinkpad
 Comment[sl]=Nastavite vmesnik KDE za posebne kontrole IBM Thinkpada
 Comment[sr]=Подесите KDE-ов интерфејс за IBM Thinkpad-ове посебне контроле
-Comment[sr@Latn]=Подесите KDE-ов интерфејс за IBM Thinkpad-ове посебне контроле
+Comment[sr@Latn]=Podesite KDE-ov interfejs za IBM Thinkpad-ove posebne kontrole
 Comment[sv]=Anpassa KDE:s gränssnitt till IBM Thinkpads specialstyrning
 Comment[ta]=ஐபிஎம் திங்க்பேட் விசேஷ கட்டுப்பாட்டிற்கு கேடிஇ இடைமுகத்தை வடிவமை
 Comment[tg]=Танзимкунии Интерфейси KDE барои Идоракунандаҳои Махсуси IBM Thinkpad
@@ -88,7 +88,7 @@
 Keywords[sk]=IBM, Thinkpad,jas,potichu
 Keywords[sl]=IBM, Thinkpad, svetlost, tiho, Jog Dial
 Keywords[sr]=Sony, Vaio, kvaio, brightness, mute, Jog Dial, светлина
-Keywords[sr@Latn]=Sony, Vaio, kvaio, brightness, mute, Jog Dial, светлина
+Keywords[sr@Latn]=Sony, Vaio, kvaio, brightness, mute, Jog Dial, svetlina
 Keywords[sv]=IBM, Thinkpad, ljusstyrka, tyst, rullhjul
 Keywords[ta]=ஐபிஎம், திங்க்பேட்,ஒளித்தன்மை,ஓசையற்ற,ஜோக் டையல்
 Keywords[tg]=IBM, Thinkpad, равшанӣ, хомӯшкунӣ, Jog Dial
diff -urN -x CVS kdeutils.orig/kmilo/thinkpad/kmilo_thinkpad.desktop kdeutils/kmilo/thinkpad/kmilo_thinkpad.desktop
--- kdeutils.orig/kmilo/thinkpad/kmilo_thinkpad.desktop	Fri Oct  1 15:54:50 2004
+++ kdeutils/kmilo/thinkpad/kmilo_thinkpad.desktop	Mon Oct  4 15:27:02 2004
@@ -25,7 +25,7 @@
 Name[sk]=Modul pre Thinkpad
 Name[sl]=Vstavek za Thinkpad
 Name[sr]=Thinkpad прикључак
-Name[sr@Latn]=Thinkpad прикључак
+Name[sr@Latn]=Thinkpad priključak
 Name[sv]=Thinkpad-insticksprogram
 Name[ta]=திங்க்பேட் செருகி
 Name[tg]=Модули Thinkpad
@@ -58,7 +58,7 @@
 Comment[sk]=Zapína podporu pre špeciálne klávesy notebookov Thinkpad
 Comment[sl]=Omogoči podporo posebnim tipkam Thinkpada
 Comment[sr]=Омогућава подршку за посебне Thinkpad-ове тастере
-Comment[sr@Latn]=Омогућава подршку за посебне Thinkpad-ове тастере
+Comment[sr@Latn]=Omogućava podršku za posebne Thinkpad-ove tastere
 Comment[sv]=Aktiverar stöd för speciella Thinkpad-tangenter
 Comment[ta]=விசேஷ திங்க்பேட் விசைகளுக்கான ஆதரவை செயல்படுத்துகிறது
 Comment[tg]=Даргиронидани пуштибонӣ барои калидҳои махсуси Thinkpad
diff -urN -x CVS kdeutils.orig/ksim/library/common.cpp kdeutils/ksim/library/common.cpp
--- kdeutils.orig/ksim/library/common.cpp	Tue Jul 22 16:41:58 2003
+++ kdeutils/ksim/library/common.cpp	Mon Oct 18 13:19:32 2004
@@ -54,6 +54,7 @@
 {
   if (!KSim::BaseList::remove(this))
     kdError() << "While trying to remove " << this << " from the list" << endl;
+  delete d;
 }
 
 bool KSim::Base::isThemeConfigOnly() const
diff -urN -x CVS kdeutils.orig/ksim/monitors/cpu/cpu.desktop kdeutils/ksim/monitors/cpu/cpu.desktop
--- kdeutils.orig/ksim/monitors/cpu/cpu.desktop	Wed Sep  1 16:19:16 2004
+++ kdeutils/ksim/monitors/cpu/cpu.desktop	Mon Oct  4 15:27:04 2004
@@ -57,7 +57,7 @@
 Comment[sk]=Modul pre monitorovanie CPU
 Comment[sl]=Vstavek za opazovanje stanja CPE
 Comment[sr]=Прикључак за надгледање CPU-а
-Comment[sr@Latn]=Dodatak za nadgledanje CPU-a
+Comment[sr@Latn]=Priključak za nadgledanje CPU-a
 Comment[sv]=Insticksprogram för processoraktivitet
 Comment[ta]= சிபியு கண்காணி சொருகுப்பொருள் 
 Comment[tg]=Модули Дидабони ВПМ
diff -urN -x CVS kdeutils.orig/ksim/monitors/disk/disk.desktop kdeutils/ksim/monitors/disk/disk.desktop
--- kdeutils.orig/ksim/monitors/disk/disk.desktop	Wed Sep  1 16:19:16 2004
+++ kdeutils/ksim/monitors/disk/disk.desktop	Tue Oct  5 15:21:07 2004
@@ -34,7 +34,7 @@
 Comment[sk]=Modul pre monitorovanie disku
 Comment[sl]=Vstavek za opazovanje diska
 Comment[sr]=Прикључак за надгледање диска
-Comment[sr@Latn]=Dodatak za nadgledanje diska
+Comment[sr@Latn]=Priključak za nadgledanje diska
 Comment[sv]=Insticksprogram för diskövervakning
 Comment[ta]= வட்டு கண்காணி சொருகுப்பொருள் 
 Comment[tg]=Модули Дидабони Диск
@@ -51,6 +51,7 @@
 Name=Disk
 Name[af]=Disket
 Name[bg]=Дискове
+Name[br]=Pladenn
 Name[ca]=Disc
 Name[el]=Δίσκος
 Name[eo]=Disko
diff -urN -x CVS kdeutils.orig/ksim/monitors/i8k/i8k.desktop kdeutils/ksim/monitors/i8k/i8k.desktop
--- kdeutils.orig/ksim/monitors/i8k/i8k.desktop	Wed Sep  1 16:19:17 2004
+++ kdeutils/ksim/monitors/i8k/i8k.desktop	Mon Oct  4 15:27:05 2004
@@ -28,7 +28,7 @@
 Comment[sk]=Modul pre monitorovanie hardware Dell I8K
 Comment[sl]=Vstavek za opazovanje strojne opreme Dell I8K
 Comment[sr]=Прикључак за надгледање Dell-овог I8K хардвера
-Comment[sr@Latn]=Dodatak za nadgledanje Dell-ovog I8K hardvera
+Comment[sr@Latn]=Priključak za nadgledanje Dell-ovog I8K hardvera
 Comment[sv]=Insticksprogram för Dell I8K-övervakning
 Comment[ta]=டெல்I8K வன்பொருள் கண்காணி சொருகுபொருள்
 Comment[tg]=Модули Дидабони Сахтафзори Dell I8K
diff -urN -x CVS kdeutils.orig/ksim/monitors/mail/mail.desktop kdeutils/ksim/monitors/mail/mail.desktop
--- kdeutils.orig/ksim/monitors/mail/mail.desktop	Wed Sep  1 16:19:18 2004
+++ kdeutils/ksim/monitors/mail/mail.desktop	Mon Oct  4 15:27:05 2004
@@ -34,7 +34,7 @@
 Comment[sk]=Modul pre monitorovanie pošty
 Comment[sl]=Vstavek za opazovanje pošte
 Comment[sr]=Прикључак за надгледање поште
-Comment[sr@Latn]=Dodatak za nadgledanje pošte
+Comment[sr@Latn]=Priključak za nadgledanje pošte
 Comment[sv]=Insticksprogram för e-postövervakning
 Comment[ta]= அஞ்சல் கண்காணி சொருகுப்பொருள் 
 Comment[tg]=Модули Дидабони Пост
diff -urN -x CVS kdeutils.orig/ksim/monitors/net/Net.desktop kdeutils/ksim/monitors/net/Net.desktop
--- kdeutils.orig/ksim/monitors/net/Net.desktop	Wed Sep  1 16:19:18 2004
+++ kdeutils/ksim/monitors/net/Net.desktop	Tue Oct  5 15:21:07 2004
@@ -49,6 +49,7 @@
 Name=Net
 Name[ar]=الشبكة
 Name[bg]=Мрежа
+Name[br]=Rouedad
 Name[ca]=Xarxa
 Name[cs]=Síť
 Name[cy]=Rhwyd
diff -urN -x CVS kdeutils.orig/ktimer/ktimer.desktop kdeutils/ktimer/ktimer.desktop
--- kdeutils.orig/ktimer/ktimer.desktop	Fri Oct  1 15:54:54 2004
+++ kdeutils/ktimer/ktimer.desktop	Mon Oct  4 15:27:07 2004
@@ -22,7 +22,7 @@
 GenericName[sk]=Odpočítavané spúšťanie
 GenericName[sl]=Zaganjalec odštevanja
 GenericName[sr]=Лансер одбројавања
-GenericName[sr@Latn]=Лансер одбројавања
+GenericName[sr@Latn]=Lanser odbrojavanja
 GenericName[sv]=Nerräknande startprogram
 GenericName[ta]= குறிப்பு எடுப்பான் நேரக்குறைப்பான்
 GenericName[tg]=Китобчаи Навиштаҷот
